index.js

```diff
@@ -156,4 +157,14 @@ client.on("message", message => {
 	}
 });
 
+client.on("messageDelete", message => {	
+	console.log("message deletion");
+	if (message.channel.name == config.counting.channelName) {
+		counting.deletion(message);
+		console.log("message deletion in counting");
+	}
+	console.log("message deletion");
```

Comment:
```suggestion
	if (message.channel.name == config.counting.channelName) {
		counting.deletion(message);
	}
```
I'm pretty sure these debug logs aren't supposed to be here.

Reply:
true